### PR TITLE
Update CVE Modal fields and Issues output

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/KFLUXBUGS-1203
https://issues.redhat.com/browse/KFLUXBUGS-1198

## Description

- Updates Modal for CVE inout
- Changes structure for bugs and cves in Release data

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

ScreenRecording2024-06-10at6 49 24PM-ezgif com-video-to-gif-converter (4)
ScreenRecording2024-06-10at6 49 24PM-ezgif com-video-to-gif-converter (3)
ScreenRecording2024-06-10at6 49 24PM-ezgif com-video-to-gif-converter (2)

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/ImportForm/__tests__/GitImportForm.spec.tsx [hunk 1]
```diff
@@ -66,8 +66,6 @@ describe('GitImportForm', () => {
     routerRenderer(<GitImportForm applicationName={undefined} />);
     expect(screen.getByText('Create application')).toBeDisabled();
     fireEvent.input(screen.getByPlaceholderText('Enter name'), { target: { value: 'test-app' } });
-    expect(screen.getByText('Create application')).not.toBeDisabled();
```
Comment (rohitkrai03) [inline src/components/ImportForm/__tests__/GitImportForm.spec.tsx:69]: Why are you changing a test for import form when there's no change related to import in this PR?

File: src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx [hunk 2]
```diff
@@ -42,23 +40,11 @@ const CVEFormContent: React.FC<CVEFormContentProps> = ({ modalToggle }) => {
           </TextContent>
         </StackItem>
         <StackItem>
-          <InputField data-test="cve-issue-key" label="CVE key" name="key" required />
+          <InputField data-test="cve-issue-key" label="CVE ID" name="key" required />
```
Comment (rohitkrai03) [inline src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx:43]: Shouldn't you also change the name and test id?

File: src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx [hunk 3]
```diff
@@ -42,23 +40,11 @@ const CVEFormContent: React.FC<CVEFormContentProps> = ({ modalToggle }) => {
...
         </StackItem>
         <StackItem>
           <ComponentField name="components" />
         </StackItem>
-        <StackItem>
-          <InputField data-test="cve-url" label="URL for the CVE" name="url" required />
```
Comment (rohitkrai03) [inline src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx:51]: We would still need this field just renamed to source, right?

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: src/components/ImportForm/__tests__/GitImportForm.spec.tsx (hunk 1 above)
Comment (abhinandan13jan) [reply src/components/ImportForm/__tests__/GitImportForm.spec.tsx]: removed changes

File: src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx (hunk 2 above)
Comment (abhinandan13jan) [reply src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx]: Updated

File: src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx (hunk 3 above)
Comment (abhinandan13jan) [reply src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/CVEFormContent.tsx:51]: No, As per the new designs, we are ommitting source and adding packages field

**Review by rohitkrai03 — APPROVED** [review]
/lgtm

**Pull request merged**